.editorconfig

```diff
@@ -5,4 +5,5 @@ charset = utf-8
 trim_trailing_whitespace = true
 insert_final_newline = true
 indent_style = spaces
-indent_size = 2
\ No newline at end of file
+indent_size = 2
+editor.detectIndentation = false
```

Comment:
What does this do? Can't find docs for this on editorconfig's site.

Reply:
😬 that's a vs code config, sorry